cluster/gce/manifests/etcd.manifest

```diff
@@ -36,6 +36,9 @@
       },
       { "name": "INITIAL_CLUSTER",
         "value": "{{ etcd_cluster }}"
+      },
+      { "name": "ETCD_CREDS",
+        "value": "{{ etcd_creds }}"
```

Comment:
Yes, `{{ etcd_creds }}` is a pre-existing template variable that is already used the `command` field of `etcd.manifest`, all we're doing here is passing it in as an environment variable as well so it can be used by the migrate script.

Reply:
OK sorry, I missed that.